test/integration/challenge_display_modes_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
+
+class ChallengeDisplayModesTest < ActionDispatch::IntegrationTest
+    include TeamsHelper
+    include Devise::Test::IntegrationHelpers
+
+    test "sponsor exists on challenge page" do
+        game = create(:active_game, board_layout: :jeopardy)
+        # Challenges are displayed by their number of categories, and then sorted
+        # down by point value
+        challenge1 = create(:standard_challenge,
+            point_value: 100,
```

Comment:
If this is only for testing the sponsor block then we should get rid of the `point_value` and `category_count` and just use the defaults here.